pkg/auto/meteremail/config/root.go

```diff
@@ -51,20 +51,22 @@ type Source struct {
 	Subsystem string `json:"subsystem,omitempty"`
 }
 
+type Timing struct {
+	Timeout      jsontypes.Duration `json:"timeout,omitempty"`
+	BackoffStart jsontypes.Duration `json:"backoffStart,omitempty"`
+	BackoffMax   jsontypes.Duration `json:"backoffMax,omitempty"`
+	NoRetries    int                `json:"noRetries,omitempty"`
```

Comment:
Can you rename this to `NumRetries` or `MaxRetries` - _NoRetries_ reads like "there will be no retries" to me